_data/hosting_providers.json

```
@@ -1,4 +1,14 @@
 [
+  {
+    "name": "interhost.co.il",
+    "link": "https://www.interhost.co.il",
+    "category": "full",
```

I think this should be partial based on our descriptions at https://certbot.eff.org/hosting_providers/ and the tutorial you included.
```suggestion
    "category": "partial",
```